### DISCUSSION TITLE
score value in the CSV file and its interpretation

### DISCUSSION THREAD

**Discussion by alirza67** [talmolab/sleap#1851] [Help!]
Hi there,

We are using SLEAP to track rats and are pleased with the performance. However, we want to filter the data and discard the keypoints that were not predicted well for the post-analysis. In this case, can we use the  'score' values in the csv files?
Can someone please explain what this score is and how we can interpret it? 

Any help is appreciated.

**Comment by talmo** [comment]
Hi @alirza67,

Apologies for the long delay!

The `instance.score` column corresponds to the detection score, which varies depending on the model type you used. For top-down, this is the centroid detection score while for bottom-up it's an aggregate measure of the overall connectivity score of the reconstructed skeleton.

The `{node_name}.score` column is the landmark localization confidence for that particular keypoint. SLEAP replaces ones that are below 0.2 (by default) with NaNs.

Let us know if you have any questions!

Cheers,

Talmo
